testkit/src/main/java/akka/kafka/testkit/javadsl/TestcontainersKafkaTest.java

```diff
@@ -22,23 +23,38 @@
  */
 public abstract class TestcontainersKafkaTest extends KafkaTest {
```

Comment:
How do users select between per-test-run and per-class Kafka brokers with this?

Reply:
For per class they would include the `void afterClass()` implementation to stop the cluster and for all tests they would omit it.  I created an implementation for the docs (`TestkitTestcontainersTest`) that demonstrates the former, but I'll add a comment to indicate how it can be used for all tests too.